lib/context_adapter_communication.js

```diff
@@ -32,21 +32,22 @@ var caLogger = require('logops'),
 
 /**
  * Sends a queryContext request to the configured Context Broker
- * @param {string} service Service header to be used in the request
- * @param {string} servicePath Service Path header to be used in the request
+ * @param {object} request The original HTTP request
  * @param {object} payload Payload to be included in the request
  * @param {function} callback Callback to be called informing of any errors or returning
  *  the obtained response
  */
-function sendQueryContext(service, servicePath, payload, callback) {
+function sendQueryContext(request, payload, callback) {
   var requestOptions = {
     method: 'POST',
     uri: 'http://' + caConfig.CB_HOST + ':' + caConfig.CB_PORT + caConfig.CB_PATH + '/queryContext',
     headers: {
       'Accept': 'application/json',
       'Content-Type': 'application/json',
-      'Fiware-Service': service || caConfig.SERVICE,
-      'Fiware-ServicePath': servicePath || caConfig.SERVICE_PATH
+      'Unica-Correlator': caHelper.getUnicaCorrelator(request),
+      'Authorization': caHelper.getTransactionId(request),
```

Comment:
As far as I understand, the mechanism to transport transaction ID is yet under discussion. Thus, I'd recomment to put this PR on hold while the mechanism gets clarified.